classes/class-wc-klarna-order-management-order-lines.php

```diff
@@ -172,6 +172,27 @@ public function process_order_line_items() {
 			}
 		}
 
+		/**
+		 * PW WooCommerce Gift Cards.
+		 */
+		foreach ( $order->get_items( 'pw_gift_card' ) as $gift_card ) {
+			$gift_card_sku    = $gift_card->get_name();
```

Comment:
Since we have a filter for this in KP, we should make sure that we use the same filter and value for the order management, else we will get warnings in the Klarna portal i suspect.